test/data/form-dataset-diff.js

```diff
@@ -9,15 +9,15 @@ export const formDatasetDiffs = dataStore({
   factory: ({
     properties
   }) => ({
-    name: faker.hacker.noun(),
+    name: faker.random.alphaNumeric(10),
     properties: properties.map(p => {
       switch (p) {
         case Property.InFormProperty:
-          return { name: faker.hacker.noun(), inForm: true };
+          return { name: faker.random.alphaNumeric(10), inForm: true };
```

Comment:
I first tried `faker.internet.userName()`, but the test still failed because that can return a string that contains `.`. I then tried `faker.random.alpha()`, but the version of Faker that we're on doesn't include `alpha()`. I landed on `alphaNumeric()` because it seems pretty close and because it allows the test to consistently pass.